test/samples/solidity/getters.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.8.0;
+
+interface Some {}
+
+contract AccessorReturns {
+    enum E {
+        A, B, C
+    }
+
+    struct S1 {
+        E e;
+        uint[] t;
+        bytes1 f;
+    }
+
+    struct S2 {
+        uint[] a;
+        address b;
+    }
+
+    struct S {
+        uint[] a;
+        int8 b;
+        string x;
+        S1 s1;
```

Comment:
@cd1m0 This is very interesting edge-case. Nested tuples preserve arrays, while root structs do not.

Reply:
Yep. I also noticed this one